### ISSUE TITLE
License issues ?

### ISSUE THREAD

**Issue by Shallowmallow** [festvox/datasets-CMU_Wilderness#4]
Awesome resources !

But I wonder if I compile the languages , will there be a license issue if I want to publish them freely on android ?

**Comment by oadams** [comment]
Yeah, you can't redistribute the language data itself; see the bible.is terms and conditions.

**Comment by Shallowmallow** [comment]
Kind of sad, would be awesome if they could accept to publish them under open source license. I tried once or twice to contact a bible website to see if I could use their data, but they never answered me.

**Comment by ssethia2** [comment]
Hey @Shallowmallow, I actually sent a message to Bible.is through their website about an issue, and they got back to me pretty quick, so you should definitely try that.
